eclipse-theia/theia
# preference: fix preference sections

#### What it does


The commit adds `toolbar` preferences to their proper section and not under `Extensions` (related to plugins). The `toolbar` preferences are now under `Features`.

![image](https://user-images.githubusercontent.com/40359487/174132657-b0bd1c8b-f23c-49e3-bc34-26f07810866c.png)


#### How to test


1. open the `preferences-view`.
2. confirm that under `Features` there are the `Toolbar` preferences.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
